worker/workercmd/worker.go

```diff
@@ -22,47 +23,92 @@ import (
 )
 
 type WorkerCommand struct {
-	Worker WorkerConfig
+	Worker WorkerConfig `yaml:",inline"`
 
-	TSA worker.TSAConfig `group:"TSA Configuration" namespace:"tsa"`
+	TSA worker.TSAConfig `yaml:"tsa,omitempty"`
```

Comment:
```suggestion
	TSA worker.TSAConfig `yaml:"gateway,omitempty"`
```

Doesn't make sense to me to call the component's config `worker_gateway` but still refer to it as `tsa` here.